kubernetes-cronjob/massdriver.yaml

```diff
@@ -5,71 +5,191 @@ source_url: github.com/YOUR_ORG/<md .Name md>
 access: private
 type: application
 
+app:
+  envs:
+    # Use jq expressions to build environment variables from input params or connections
+    LOG_LEVEL: .params.configuration.log_level
```

Comment:
Updated these to match example section that I added commented out at the bottom of params